src/components/profile-view/profile-view.jsx

```diff
@@ -41,6 +51,32 @@ export function ProfileView(props) {
             });
     };
 
+    const handleUpdateUser = (e) => {
+        e.preventDefault();
+        /* Send a request to the server for authentication */
+        axios.put('https://morning-badlands-52426.herokuapp.com/users/' + props.userData.Username,
+            userInfo,
+            {
+                headers: { Authorization: `Bearer ${token}` },
+            })
+            /* then call props.onRegistration(username) */
+            .then(response => {
+                const data = response.data;
+                console.log(data);
+                alert("user updated");
+                localStorage.setItem('user', username);
+                location.reload();
+                props.history.push("/");
```

Comment:
Not a blocking requirement but no need to reload here to get updated info. `data` contains all the updated information you need. You can fix this at your own discretion. The fix would be to call the same method you call on login to pass the updated information for storage either in-state or localstorage.

Reply:
Ok, I removed the location.reload()